### PR TITLE
Fix a bunch of typos in test/e2e/local

### PR DESCRIPTION
**Pull request by m1093782566**
cc @hklai @JimmyCYJ

### CODE REVIEW

File: tests/e2e/local/minikube/Troubleshooting.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ This file helps note down issues we have seen and how to debug them
    ```
 
    The `GOOS=linux` is required when you are running setup on macOS.
-1. If your docker push step failed, please check if you have port forwarding setup for localregistry or not and does docker know about insecure registry at localhost:5000 or not. You can run following script to do that as well.
+1. If your docker push step failed, please check if you have port forwarding setup for local registry or not and does docker know about insecure registry at localhost:5000 or not. You can run following script to do that as well.
```
Comment (hklai) [inline tests/e2e/local/minikube/Troubleshooting.md]: I think we can remove this since we do not use local register in the minikube case...

Comment (m1093782566) [reply tests/e2e/local/minikube/Troubleshooting.md]: Yes, I checked the codes and we can safely remove the docker push part in this file.

Comment (m1093782566) [reply tests/e2e/local/minikube/Troubleshooting.md]: Comments are fixed now, PTAL :)

File: tests/e2e/local/minikube/Troubleshooting.md [hunk 2]
```diff
@@ -24,7 +18,7 @@ This file helps note down issues we have seen and how to debug them
    ```
 
 1. When running tests, if you get "Bad Request" error
-   Minikube uses insecure localregistry opened at localhost:5000, make sure you specify HUB as `localhost:5000` and TAG as `latest` when running the tests.
+   Minikube uses insecure local registry opened at localhost:5000, make sure you specify HUB as `localhost:5000` and TAG as `latest` when running the tests.
```
Comment (hklai) [inline tests/e2e/local/minikube/Troubleshooting.md]: This is out of date as well. Could you please remove or update?

File: tests/e2e/local/minikube/Troubleshooting.md (hunk 1 above)
Comment (m1093782566) [reply tests/e2e/local/minikube/Troubleshooting.md]: hmmmm, let me check the codes.

File: tests/e2e/local/minikube/Troubleshooting.md (hunk 2 above)
Comment (m1093782566) [reply tests/e2e/local/minikube/Troubleshooting.md]: hmmmm, let me check the codes.

Comment (m1093782566) [reply tests/e2e/local/minikube/Troubleshooting.md]: @hklai 

After checking [minikube/setup_test.sh](https://github.com/istio/istio/blob/master/tests/e2e/local/minikube/setup_test.sh#L14-L17), I assume we still need this line?

Comment (hklai) [reply tests/e2e/local/minikube/Troubleshooting.md]: Check the line again. It is just make docker but it does not push to anywhere.

Docker just picks up the images locally. So the comment there is probably out of date too. =)

Comment (hklai) [reply tests/e2e/local/minikube/Troubleshooting.md]: Check the line again. It is just make docker but it does not push to anywhere.

Docker just picks up the images locally. So the comment there is probably out of date too. =)

Comment (m1093782566) [reply tests/e2e/local/minikube/Troubleshooting.md]: oh, just missing `make push`?

Thanks for reminding. I just fix that :) PTAL.

**Comment by hklai** [comment]
To be clear, missing push is intentional. Fix is needed in those comments and docs instead.

**Comment by m1093782566** [comment]
Yes, I understand it.

**Comment by m1093782566** [comment]
Just pushed an update in comments and docs.

**Pull request merged**